generators/project/templates/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "<%= title %>",
+    "scripts": {
+        
+        "build": "mbt build --mtar <%= title %>.mtar",
+        "build:workspaces": "npm run build --workspaces --if-present",
+        "deploy": "cf deploy mta_archives/<%= title %>.mtar --retries 0",
+		"lint": "npm run lint --workspaces --if-present",
+		"ui5lint": "npm run ui5lint --workspaces --if-present"
+    },
+    "dependencies": {
+        "mbt": "<%= mbtVersion %>"
```

Comment:
don't immediately grok where this is defined and why it needs to be a placeholder...`^1`?

Reply:
It's defined in the `dependencies.js`, where all dependencies are managed centrally.